### ISSUE TITLE
rollup-plugin-livereload can`t watch html file and reload it when html file was changed!

### ISSUE THREAD

**Issue by zhenwoo** [thgh/rollup-plugin-livereload#16]
my config as follows:
```
import babel from 'rollup-plugin-babel';
import { uglify } from "rollup-plugin-uglify";
import serve from 'rollup-plugin-serve';
import livereload from 'rollup-plugin-livereload';
import postcss from 'rollup-plugin-postcss';
import discardComments from 'cssnano-preset-default';
import copy from 'rollup-plugin-copy-assets';
import htmlTemplate from 'rollup-plugin-bundle-html'
import module from './module.config.js'
const pro = process.env.NODE_ENV == 'production';
const dev = process.env.NODE_ENV == 'development';
let app = null;
for(var i in module) {
    if (module[i].open) {
        app = module[i];
        break;
    }
}
export default {
    input:`${app.src}/js/app.js`,
    output: {
        file: `${app.dist}/js/app.js`,
        format: 'umd'
    },
    plugins: [
        dev && serve({
            contentBase: `${app.dist}/`,
            host: 'localhost'
        }),
        dev && livereload({
            exts: ['html', 'js', 'less', 'css'],
            watch: `${app.dist}/`
        }),
        postcss({
            extensions:['.less'],
            extract:  `${app.dist}/css/bundle.css`,
            minimize: pro ? discardComments : false
        }),
        copy({
            assets: [
                `${app.src}/imgs`
            ]
        }),
        htmlTemplate({
            template: `${app.src}/index.html`,
            dest: `${app.dist}`,
            filename: 'index.html',
            inject: 'body'
        }),
        babel({
            exclude: 'node_modules/**',
            plugins: ['external-helpers'],
            externalHelpers: true
        }),
        pro && uglify(),
    ]
}
```

**Comment by thgh** [comment]
Which version of rollup, rollup-plugin-livereload and rollup-plugin-serve are you using?

**Comment by zhenwoo** [comment]
> Which version of rollup, rollup-plugin-livereload and rollup-plugin-serve are you using?

this is my package.json
```
{
  "name": "rollup",
  "version": "1.0.0",
  "main": "index.js",
  "license": "MIT",
  "scripts": {
    "build": "cross-env NODE_ENV=production rollup -c",
    "start": "cross-env NODE_ENV=development rollup -c --watch"
  },
  "devDependencies": {
    "babel-core": "^6.26.3",
    "babel-helper-explode-class": "^6.24.1",
    "babel-plugin-external-helpers": "^6.22.0",
    "babel-preset-env": "^1.7.0",
    "cross-env": "^5.2.0",
    "cssnano-preset-default": "^4.0.7",
    "less": "^3.9.0",
    "livereload": "^0.7.0",
    "rollup": "^1.4.0",
    "rollup-plugin-babel": "3",
    "rollup-plugin-bundle-html": "^0.1.4",
    "rollup-plugin-copy-assets": "^1.1.0",
    "rollup-plugin-livereload": "^1.0.0",
    "rollup-plugin-postcss": "^2.0.3",
    "rollup-plugin-serve": "^1.0.1",
    "rollup-plugin-uglify": "^6.0.2"
  }
}
```

**Comment by thgh** [comment]
Just noticed this issue myself. It seems that the plugin is reinitialized, so moving `const server` up to the module scope should allow to terminate it and start a new one.
https://github.com/thgh/rollup-plugin-livereload/blob/62699da684c2162e5784b828a2e51b3fd37fc401/src/index.js#L16

**Comment by aminnairi** [comment]
Is this issue still being worked on? I'm having a similar issue for files that are not the `input`.

**Comment by thgh** [comment]
This is probably fixed in v1.2.0, will close for now. Always welcome to update with more info!
